code/modules/antagonists/abductor/equipment/abduction_gear.dm

```diff
@@ -794,3 +794,12 @@ Congratulations! You are now trained for invasive xenobiology research!"}
 	airlock_type = /obj/machinery/door/airlock/abductor
 	material_type = /obj/item/stack/sheet/mineral/abductor
 	noglass = TRUE
+
+/obj/item/clothing/under/abductor
+	desc = "The most advanced form of jumpsuit known to reality, looks uncomfortable."
+	name = "alien jumpsuit"
+	icon = 'icons/obj/abductor.dmi'
```

Comment:
This line needs to be removed, and you need to put the obj icon in /icons/obj/clothing/uniforms.dmi

The clothing icons themselves (not the obj icon) go in /icons/mob/under.dmi

The other file you added in abductor.dmi appears to serve no purpose and should be removed.

Also you need to update your branch as it appears it didn't have the hazard suit obj icon, hence why the icon diff bot shows you removing it.

Reply:
Also, you should post some pics or a short video of how it looks ingame on the actual lads.

Reply:
Thanks for the feedback, I'm still very new to this, I'll start getting this done.